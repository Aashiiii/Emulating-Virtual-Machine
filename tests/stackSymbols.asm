FUNC LABEL 0
    MOV REG 0 VAL 10
    MOV STK A REG 0
    MOV STK B STK A
    CAL VAL 5
    CAL VAL 3
    CAL VAL 7
    PRINT STK A
    RET
FUNC LABEL 3
    MOV STK A VAL 3
    REF STK B STK A
    PRINT PTR B
    RET
FUNC LABEL 5
    MOV STK A VAL 41
    MOV STK B VAL 93
    REF STK C STK A
    PRINT PTR C
    RET
FUNC LABEL 7
    MOV REG 0 VAL 4
    MOV REG 1 VAL 3
    ADD REG 0 REG 1
    PRINT REG 0
    RET
