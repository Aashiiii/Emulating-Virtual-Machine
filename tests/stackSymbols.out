41
3
7
10
